prototype2/client/core/renderer/src/opengl/mod.rs

```diff
@@ -120,8 +120,15 @@ impl OpenGlRenderer {
                    0.0, 1.02, 0.0,  0.0,
                    0.0, 0.0, 1.02, 0.0,
                    x, y, (z + 1.0), 1.0);
-    let (rx, ry, rz) = physical_aspect.ang;
-    let rotation = Matrix4::from(Euler::new(Rad::new(-rx), Rad::new(-rz), Rad::new(-ry)));
+    // let (rx, ry, rz) = physical_aspect.ang;
+    let a = physical_aspect.ang_rich;
+    // let rotation = Matrix4::from(Euler::new(Rad::new(-rx), Rad::new(-rz),
+    // Rad::new(-ry)));
+    let rotation_3 = 
+                     //Matrix3::new(0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0);
+                    Matrix3::new(-1.0, 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, -1.0, 0.0) *
```

Comment:
This needs to be a const. Looks like the physics engine is exactly (-x, -z, -y) from the rendering. Dunno how that happened, but this matrix reflects that transformation (as did the old mat4 above).